services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -2290,6 +2290,12 @@ public synchronized String allowOutgoingOnPrivate(String destCidr) {
         Script command = new Script("/bin/bash", s_logger);
         String intf = "eth1";
         command.add("-c");
+        command.add("iptables -D OUTPUT -o " + intf + " -d " + destCidr + " -p tcp -m state --state NEW -m tcp  -j ACCEPT");
```

Comment:
@DaanHoogland 
would be better to check by `iptables-save |grep "<command>"` ?
the command looks like "OUTPUT -o xxxx"

Reply:
@DaanHoogland 
my idea is
- check if the rule exists by command `iptables-save |grep "<command>"`
- if exist, do nothing and return
- if not exist, insert the rule

it looks better than `-D` and then `-I`

Reply:
yes, agreed. I don't like the grep though, that's why I wonder if we can use the `-C` flag for this. It seems made for exactly this purpose.

Reply:
@DaanHoogland 
yeah, `-C` is even better if it works.

Reply:
ok, will try to implement that